src/displayBackend/common/drm_edid.h

```diff
@@ -42,9 +41,9 @@ struct i2c_adapter;
 #define DISPLAYID_EXT 0x70
 
 struct est_timings {
-	u8 t1;
-	u8 t2;
-	u8 mfg_rsvd;
+	__u8 t1;
```

Comment:
Why not use uint8_t from <cstdint>? According to https://stackoverflow.com/questions/16232557/difference-between-u8-and-uint8-t uint8_t is more portable.

Reply:
ok